explainaboard/metric.py

```diff
@@ -151,7 +152,13 @@ def aggregate_stats(self, stats: MetricStats) -> np.ndarray:
         :param stats: stats for every example
         :return: aggregated stats
         """
-        return np.mean(stats.get_data(), axis=0)
+
+        res = copy.deepcopy(stats)
+
+        if res.get_data().size == 0:
+            return np.array(0.0)
```

Comment:
I'm assuming this is a special case for buckets where the number of examples is zero.

As an alternative suggestion: could we make this `np.zeros(self.stats_length)` where `self.stats_length` is the expected number of columns the sufficient statistics? This would reduce the necessity to make special cases throughout the rest of the code.

Reply:
aha, I'm not sure if I completely get your point so I leave this un-implemented first.